internal/apiserver/clusterservice/clusterimpl.go

```diff
@@ -1909,6 +1972,34 @@ func UpdateCluster(request *msgs.UpdateClusterRequest) msgs.UpdateClusterRespons
 			cluster.Spec.BackrestResources[v1.ResourceMemory] = quantity
 		}
 
+		// ensure there is a value for ExporterResources
+		if cluster.Spec.ExporterResources == nil {
+			cluster.Spec.ExporterLimits = v1.ResourceList{}
+			cluster.Spec.ExporterResources = v1.ResourceList{}
+		}
```

Comment:
On plain Kubernetes containers, limit-only implies limit == request. Does our code allow one to specify only limits? If so how does it pass through the layers?

It looks like this block might incorrectly overwrite / zero-out `ExporterLimits` if they are populated at this point when `ExporterResources` are not.

Reply:
This is matching the existing pattern for the other resource requests.